drivers/include/mtd_mapper.h

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright (C) 2020  Koen Zandberg
```

Comment:
Her and in the other files: there's an extra space. And usually we add an email address...

Reply:
> Her and in the other files: there's an extra space. And usually we add an email address...

Good catch, amended